Makefile

```diff
@@ -0,0 +1,29 @@
+BIN_DIR := hack
+MDBOOK_BIN := $(BIN_DIR)/mdbook
+MDBOOK_VERSION := v0.4.3
+MDBOOK_RELEASE_URL := https://github.com/rust-lang/mdBook/releases/download/$(MDBOOK_VERSION)/mdbook-$(MDBOOK_VERSION)-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
I don't suppose there's a container image for the binary, is there?

Reply:
There are some images available in Docker Hub, but I tried to avoid using them since they are not official ones and not recommended in the Mdbook [documentation ](https://github.com/rust-lang/mdBook).

Reply:
Maybe we could build our own? We could add a Dockerfile to this repo and set up quay.io to build it. That can all happen separately, and shouldn't block progress on these changes.

Reply:
Sure thing. I can sign up for adding a Dockerfile in a separate PR.

Reply:
Added a Dockerfile in a new PR https://github.com/metal3-io/metal3-docs/pull/158.